test/options.test.js

```diff
@@ -19,68 +19,64 @@ const OPTION_DEFAULTS = {
 }
 
 describe('options', () => {
-  describe('defaults', () => {
-    it('should be as expected', () => {
-      return expect(optionDefaults).to.deep.equal(OPTION_DEFAULTS)
-    })
+  it('defaults should be as expected', () => {
+    expect(optionDefaults).to.deep.equal(OPTION_DEFAULTS)
   })
 
   describe('input: file', () => {
     it('should accept a file path as input', () => {
       return expect(
-        run('.', PATH_JSON_FIXTURE, {
-          input: 'file'
-        })
+        run('.', PATH_JSON_FIXTURE, { input: 'file' })
       ).to.eventually.be.fulfilled
     })
   })
 
   describe('input: json', () => {
     it('should accept a json object as input', () => {
       return expect(
-        run('.', FIXTURE_JSON, {
-          input: 'json'
-        })
+        run('.', FIXTURE_JSON, { input: 'json' })
       ).to.eventually.be.fulfilled
     })
   })
 
   describe('input: string', () => {
     it('should accept a json string as input', () => {
       return expect(
-        run('.', FIXTURE_JSON_STRING, {
-          input: 'string'
-        })
+        run('.', FIXTURE_JSON_STRING, { input: 'string' })
       ).to.eventually.be.fulfilled
     })
   })
 
   describe('output: json', () => {
-    it('should return a minified json string', () => {
+    it('should return a json', () => {
       return expect(
-        run('.', PATH_JSON_FIXTURE, {
-          output: 'json'
-        })
+        run('.', PATH_JSON_FIXTURE, { output: 'json' })
       ).to.eventually.become(FIXTURE_JSON)
     })
+
+    it('should return a json in case of filter fail', () => {
```

Comment:
This isn't obvious enough what it means. I think this should be something like `it should return a string if the filter calls an array`